Reader post content tweaks for tablets

hypest opened the pull request:
This PR introduces a few small tweaks to how the Reader displays the post content. All in all, the end result is visually closer to what the iPad app and Calypso are display.

Here's a rundown of the tweaks:

* Narrower content width when viewed on a tablet (limit width to 600dp)
* Remove line separator between the blog title (header) and the post title
* Increase the content font size a bit (CSS now set to `font-size: 100%`)
* Increase the margins between the image, its caption and the post body
* Center align the image caption with the image
* Allow for left/right aligned images with text flowing next to them
* Remove the grey background of the images container
* 

While all of the post content and info are following the "narrow" column layout, I intentionally left the bottom bar (which harbours the tag name and comments/likes counts) full-width as it serves as a toolbar.

Would be nice to have some feedback on this from a designer as well. @rickybanister, you think you can have a look?

To make reviewing a bit easier, I've prepared a couple of "before" and "after" screengrab videos (on a 9" tablet):

**Before**: https://cloudup.com/cPr4LJUagqh
**After**: https://cloudup.com/cOF1oI-lCfG


To test:

* Using a large-ish tablet (or a 7" one in landscape)
* Open a post with images in the Reader
* Notice the various visual tweaks being applied

/cc @oguzkocer , @sendhil

rickybanister commented:
It looks pretty good to me—I pinged some Reader Squad folks to also take a look, so perhaps lets give them a chance to chime in.

nbradbury commented on WordPress/src/main/res/values/attrs.xml:
```diff
@@ -82,4 +82,8 @@
         </attr>
     </declare-styleable>
 
+    <declare-styleable name="BoundedWidth">
```
Personal preference: I like to use the "wp" prefix for our own custom attributes to avoid possible collisions with third-party libraries (and even Android itself) down the road.

hypest replied:
Makes better sense so, fixed with ac1b633. Thanks!

nbradbury commented on WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderWebView.java:
```diff
@@ -78,11 +84,21 @@ public boolean isDestroyed() {
     public ReaderWebView(Context context, AttributeSet attrs) {
         super(context, attrs);
+
+        TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.BoundedWidth);
```
Perhaps we could avoid obtaining mBoundedWidth both here and in the next constructor by simply calling `this(context, attrs, 0);` here?

hypest replied:
Nice chance to reduce code repetition actually. With cf6faa4, I've opted to move that loading into `init()` instead of shortcircuiting the constructors. Thanks!

nbradbury commented:
Apart from some minor nits, I think this looks good. I can't say I like how the bottom toolbar no longer aligns with the content, but that may be simply because I'm so used to it being the other way.

hypest commented:
Thanks for the feedback @rickybanister ! Since I'm not sure exactly who you pinged for extra review, would you mind adding their handles here so to know who should I wait for? Thanks!

hypest commented:
Hey @nbradbury , I've addressed the points you raised and I think this is ready for another pass.

I see what you mean about the bottom bar aligning. FWIW, Calypso also has the relevant bar (although it's the top one in that case and without edge paddings) not aligned with the content. We could try removing the edge paddings in our bar as well and see how that goes actually. Wdyt?

blowery commented:
cc @jancavan

blowery commented:
LGTMBIANAD

blowery commented:
I kind of liked having the like and tag bits aligned with the content too. Does feel a bit odd to have them out in space.

jancavan commented:
For tags/comments/likes, I think it's better to align them with either the elements on the blue masterbar or the content. It feels like they're on their own grid now.

hypest commented:
Hey guys, I've updated the implementation to also align the bottom "tag" bar with the reader post detail content. I've kept the bar separator to full width as I think it looks better and makes it more clear that the bottom bar is "disconnected" from the content. /cc @nbradbury , @blowery , @jancavan 

While at it, I increased the maximum width a bit (to `680dp`) to closer match Calypso and also made the tag hit target more tap friendly (height to `48dp` as the Android guidelines suggest).

@nbradbury I think this is ready for another pass. Thanks!

jancavan commented:
Design-wise, the changes LGTM 👍

nbradbury commented:
@hypest Sorry this got prolonged due to my vacationing! The latest changes look good, but it seems Travis is stuck for some reason. If you can resolve whatever's bugging Travis I'm good to merge.

nbradbury commented:
:shipit:
